pkg/util/tracing/tracer.go

```diff
@@ -520,6 +520,10 @@ func (t *Tracer) startSpanGeneric(
 	helper.crdbSpan.mu.operation = opName
 	helper.crdbSpan.mu.recording.logs = newSizeLimitedBuffer(maxLogBytesPerSpan)
 	helper.crdbSpan.mu.recording.structured = newSizeLimitedBuffer(maxStructuredBytesPerSpan)
+	helper.crdbSpan.mu.tags = helper.crdbSpan.mu.tagsAlloc[:0]
```

Comment:
Instead of this `tagsAlloc` guy sitting on `crdbSpanMu`, you could just have it as part of the `helper` struct above, right?